src/aliceVision/camera/CMakeLists.txt

```diff
@@ -24,6 +33,7 @@ alicevision_add_interface(aliceVision_camera
 )
 
 # Unit tests
+alicevision_add_test(distortion_test.cpp NAME "camera_distortionRadial"    LINKS aliceVision_camera)
```

Comment:
(cosmetics) align NAME with lines 37-39

Reply:
yes my bad, i changed the name a couple of times and the formatting lost consistency, fixed